src/domains/transaction/validations/SendTransfer.ts

```diff
@@ -0,0 +1,40 @@
+export const sendTransfer = (t: any) => ({
+	fee: () => ({
+		required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
+			field: t("COMMON.FEE"),
+		}),
+	}),
+	senderAddress: () => ({
+		required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
+			field: t("COMMON.SENDER_ADDRESS"),
+		}),
+	}),
+	smartbridge: () => ({
+		required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
+			field: t("COMMON.SMARTBRIDGE"),
+		}),
```

Comment:
```suggestion
```

Reply:
The smartbridge is an optional field :) don't worry about that <div>, I'll fix it in a separate PR as there are more adjustments needed 👍

Reply:
Was confused due to the fact that I saw the required dot :-)